menu/menu_displaylist.c

```diff
@@ -12344,7 +12482,7 @@ bool menu_displaylist_ctl(enum menu_displaylist_ctl_state type,
 #endif
 #endif
 
-#ifdef HAVE_COMPRESSION
+#if defined(HAVE_COMPRESSION) && !defined(HAVE_MIST)
```

Comment:
Why remove this for `HAVE_MIST` builds? Core system files are still needed on Steam...

Reply:
Due to agreement, but looking back it I will probably have to ditch the entire online updater and move the core manager up to the main menu.

Reply:
That's fair enough.

I suppose a mechanism will need to be added for core system files to be installed automatically when a core is installed...